doc/release-process.md

```diff
@@ -1,29 +1,99 @@
-How to build release binary
-==============================
+Release Process
+===============
+
+See also [Tapyrus Core Release Process](https://github.com/chaintope/tapyrus-core/blob/master/doc/release-process.md). In general, Tapyrus Signer and Tapyrus Core will be released at the same 
+time.
+
+Before every minor and major release:
+
+* Update version in `Cargo.toml`. See also [vertioning rule](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/versioning_rule.md).
+* Write release notes (see below)
+* Update Testnet environment. Tapyrus Testnet environment documents are [here](https://drive.google.com/drive/folders/1-J6GE77dH2u5giKoNsuQr24v02bQJjZp) (This is internal document in Chaintope Inc.)
```

Comment:
I think, link to Google drive does not necessary.

Reply:
fixed!